src/rockstor/scripts/prep_db.py

```diff
@@ -37,12 +37,20 @@ def register_services():
         'NUT-UPS': 'nut',
         'ZTaskd': 'ztask-daemon',
         'Bootstrap': 'rockstor-bootstrap',
-        'Rockstor': 'rockstor',}
+        'Shell In A Box': 'shellinaboxd',
+        'Rockstor': 'rockstor'
+        }
+
+    services_configs = {
```

Comment:
+1 for this idea. After leaving my comments I was thinking that it may be better UX if we just assume default config and start the daemon if it's never been configured before and the user happens to access the shell via web-ui. But this method works too. Feel free to improve it further in a future PR. As far as I am concerned, this is impressive as is, very useful feature!